runtime/scripts.vim

```diff
@@ -245,7 +245,7 @@ else
     set ft=xhtml
 
     " HTML (e.g.: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN")
-  elseif s:line1 =~? '\<DOCTYPE\s\+html\>'
+  elseif s:line1 =~? '\<<!DOCTYPE\s\+html\>'
```

Comment:
@k-takata I think it is. Otherwise we might match a line starting with `abc<!doctype` and we don't want that. ´\<´ is treated as a word boundary, right?